syntaxes/Dafny.tmLanguage

```diff
@@ -142,7 +142,7 @@
 				</dict>
 			</dict>
 			<key>end</key>
-			<string>(?=&lt;|\s*|,|\)|\}|(?:requires|ensures|modifies|reads|class|codatatype|const|constructor|datatype|function|greatest|iterator|least|lemma|method|module|newtype|predicate|trait|type|var)\b))</string>
+			<string>(?=&lt;|\s*|,|\)|\}|(?:requires|ensures|modifies|reads|class|codatatype|const|constructor|datatype|function|greatest|iterator|least|lemma|method|module|newtype|opaque|predicate|trait|type|var)\b))</string>
```

Comment:
ditto

Reply:
```suggestion
			<string>(?=&lt;|\s*|,|\)|\}|(?:requires|ensures|modifies|reads|class|codatatype|const|constructor|datatype|function|greatest|iterator|least|lemma|method|module|newtype|opaque|predicate|trait|type|var|abstract|static|ghost)\b))</string>
```

Reply:
Here as well, it's only a list of keywords to stop the parsing of variable: type